### DISCUSSION TITLE
geometryType: ANY

### DISCUSSION THREAD

**Discussion by ghelle-fhh** [ldproxy/ldproxy#804] [Configuration]
Hi,

I have a small question about the property _geometryType_

Are there any disadvantages/issues in using geometryType: ANY compared to explicitly listing the correct geometry (i.e. geometryType: LINE_STRING or geometryType: MULTI_POLYGON)?

For example, would it be problematic to simply use geometryType: ANY as a hardcoded value in all provider configurations - instead of explicitly listing the actual geometry type?

Thanks!

**Comment by cportele** [comment]
Hi @ghelle-fhh, I think the geometryType information in the provider schema is - at least at the moment - only used for metadata that informs clients/users. This includes feature schemas, queryables, tileset metadata, etc. That is, it will depend on how the data is used / processed if it makes a difference.

There may be a few exceptions. For example, to provide solid geometries in CityJSON or JSON-FG the geometry type of the property must be a MULTI_POLYGON (that is closed and composite).
